src/components/atoms/Flex/index.tsx

```diff
@@ -1,10 +1,12 @@
-import { ReactNode, CSSProperties } from "react";
+import { ReactNode, CSSProperties, AriaRole } from "react";
 
 export type Props = {
   className?: string;
   onClick?: () => void;
   children?: ReactNode;
   testId?: string;
+  role?: AriaRole;
+  hidden?: boolean;
```

Comment:
Should these be `ariaRole` and `ariaHidden`? I'm not sure these are clear enough as is. Especially hidden sounds like I can hide the Flex component and its contents by passing true to there.